app/templates/_proposition_header.html

```diff
@@ -4,12 +4,12 @@
     <nav id="proposition-menu">
       <a href="/" id="proposition-name">Digital Marketplace</a>
       <ul id="proposition-links">
-        {% if current_user.email_address %}
-        <li><a class="home" href="/logout">Log out</a></li>
+        {% if current_user.is_authenticated() %}
+          <li><a href="{{ url_for('main.dashboard') }}">View your account</a></li>
+          <li><a class="home" href="/logout">Log out</a></li>
```

Comment:
What does the "home" class give us here - should the "View your account" link get it too?